tests/Doctrine/Tests/DBAL/Cache/QueryCacheProfileTest.php

```diff
@@ -5,65 +5,61 @@
 use Doctrine\DBAL\Cache\QueryCacheProfile;
 use Doctrine\DBAL\ParameterType;
 use Doctrine\Tests\DbalTestCase;
+use function parse_str;
+use function strpos;
 
 class QueryCacheProfileTest extends DbalTestCase
 {
-    const LIFETIME = 3600;
-    const CACHE_KEY = 'user_specified_cache_key';
+    public const LIFETIME  = 3600;
+    public const CACHE_KEY = 'user_specified_cache_key';
 
     /** @var QueryCacheProfile */
     private $queryCacheProfile;
 
-    protected function setUp()
-    {
-        $this->queryCacheProfile = new QueryCacheProfile(self::LIFETIME, self::CACHE_KEY);
-    }
+    /** @var string */
+    private $query = 'SELECT * FROM foo WHERE bar = ?';
 
-    public function testShouldUseTheGivenCacheKeyIfPresent()
-    {
-        $query  = 'SELECT * FROM foo WHERE bar = ?';
-        $params = [666];
-        $types  = [ParameterType::INTEGER];
+    /** @var array<int> */
+    private $params = [666];
 
-        $connectionParams = array(
+    /** @var array<string> */
+    private $types = [ParameterType::INTEGER];
+
+    /** @var array<string> */
+    private $connectionParams = [
```

Comment:
Alignment went a bit bonkers here. Also, the type is `string[]`, as most tooling doesn't work with `array<foo>`